docs/standard/microservices-architecture/architect-microservice-container-applications/direct-client-to-microservice-communication-versus-the-API-Gateway-pattern.md

```diff
@@ -116,13 +115,13 @@ For more information, see [Gateway offloading pattern](https://docs.microsoft.co
 
 ## Using products with API Gateway features
 
-There can be many more cross-cutting concerns offered by the API Gateways products depending on each implementation. For instance, [Azure API Management](https://azure.microsoft.com/services/api-management/) (as shown in Figure 4-14) not only solves your API Gateway needs but provides features like gathering insights from your APIs. If you are using an API management solution, an API Gateway is only a component within that full API management solution.
+There can be many more cross-cutting concerns offered by the API Gateways products depending on each implementation. For instance, **[Azure API Management](https://azure.microsoft.com/services/api-management/)** (as shown in Figure 4-14) not only solves your API Gateway needs but provides features like gathering insights from your APIs. If you are using an API management solution, an API Gateway is only a component within that full API management solution.
```

Comment:
don't style links

Reply:
Is it OK to leave the italics style in the "Additional Resources" sections?